Error running tests on NixOS

I'm trying to package `carapace` 0.12.3 for NixOS, but I get this:

```
== RUN   TestActionDirectories
    assert.go:21:
        1c1
        < {Action:{rawValues:[{Value:docs/ Display:docs/ Description:} {Value:example/ Display:example/ Description:} {Value:internal/ Display:internal/ Description:} {Value:pkg/ Display:pkg/ Description:}] callback:<nil> nospace:true skipcache:false}}
        ---
        > {Action:{rawValues:[{Value:docs/ Display:docs/ Description:} {Value:example/ Display:example/ Description:} {Value:internal/ Display:internal/ Description:} {Value:pkg/ Display:pkg/ Description:} {Value:vendor/ Display:vendor/ Description:}] callback:<nil> nospace:true skipcache:false}}

    assert.go:21:
        1c1
        < {Action:{rawValues:[{Value:./docs/ Display:docs/ Description:} {Value:./example/ Display:example/ Description:} {Value:./internal/ Display:internal/ Description:} {Value:./pkg/ Display:pkg/ Description:}] callback:<nil> nospace:true skipcache:false}}
        ---
        > {Action:{rawValues:[{Value:./docs/ Display:docs/ Description:} {Value:./example/ Display:example/ Description:} {Value:./internal/ Display:internal/ Description:} {Value:./pkg/ Display:pkg/ Description:} {Value:./vendor/ Display:vendor/ Description:}] callback:<nil> nospace:true skipcache:false}}

--- FAIL: TestActionDirectories (0.00s)
=== RUN   TestActionFiles
    assert.go:21:
        1c1
        < {Action:{rawValues:[{Value:README.md Display:README.md Description:} {Value:docs/ Display:docs/ Description:} {Value:example/ Display:example/ Description:} {Value:internal/ Display:internal/ Description:} {Value:pkg/ Display:pkg/ Description:}] callback:<nil> nospace:true skipcache:false}}
        ---
        > {Action:{rawValues:[{Value:README.md Display:README.md Description:} {Value:docs/ Display:docs/ Description:} {Value:example/ Display:example/ Description:} {Value:internal/ Display:internal/ Description:} {Value:pkg/ Display:pkg/ Description:} {Value:vendor/ Display:vendor/ Description:}] callback:<nil> nospace:true skipcache:false}}
```

My `go` is not good enough to understand the problem, though. Can you help?

--- a/action_test.go
+++ b/action_test.go
@@ -103,12 +103,12 @@ func TestNoSpace(t *testing.T) {
 func TestActionDirectories(t *testing.T) {
 	assertEqual(t,
 		ActionValues("example/", "docs/", "internal/", "pkg/").noSpace(true).Invoke(Context{}),
-		ActionDirectories().Invoke(Context{CallbackValue: ""}),
+		ActionDirectories().Invoke(Context{CallbackValue: ""}).Filter([]string{"vendor/"}),
 	)
 
 	assertEqual(t,
 		ActionValues("example/", "docs/", "internal/", "pkg/").noSpace(true).Invoke(Context{}).Prefix("./"),
-		ActionDirectories().Invoke(Context{CallbackValue: "./"}),
+		ActionDirectories().Invoke(Context{CallbackValue: "./"}).Filter([]string{"./vendor/"}),
 	)
 
 	assertEqual(t,
@@ -125,7 +125,7 @@ func TestActionDirectories(t *testing.T) {
 func TestActionFiles(t *testing.T) {
 	assertEqual(t,
 		ActionValues("README.md", "example/", "docs/", "internal/", "pkg/").noSpace(true).Invoke(Context{}),
-		ActionFiles(".md").Invoke(Context{CallbackValue: ""}),
+		ActionFiles(".md").Invoke(Context{CallbackValue: ""}).Filter([]string{"vendor/"}),
 	)
 
 	assertEqual(t,

